src/LaravelDoctrineSanctumProvider.php

```diff
@@ -28,11 +28,18 @@
 
 class LaravelDoctrineSanctumProvider extends ServiceProvider
 {
+    public function register(): void
+    {
+        $this->mergeConfigFrom(
+            __DIR__.'/../config/sanctum_orm.php', 'sanctum_orm'
+        );
+    }
+
     public function boot(): void
     {
         $this->publishes([
-            __DIR__.'/Config/sanctum.php' => config_path('sanctum.php'),
-        ]);
+            __DIR__.'/../config/sanctum_orm.php' => config_path('sanctum_orm.php'),
```

Comment:
Is there a specific reason why you renamed the config from `sanctum` to `sanctum_orm`. The word ORM gives me another representation of what sanctum does.